demo/app/lib/widgets/success_card.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:app/models/credential_data.dart';
+import 'package:flutter/material.dart';
+import 'package:app/widgets/credential_card.dart';
+
+class SuccessCard extends StatefulWidget {
+  String? verifierName;
+  String? subTitle;
+  CredentialData credentialData;
+
+  SuccessCard({required this.credentialData, this.verifierName, this.subTitle,  Key? key}) : super(key: key);
+
+  @override
+  State<SuccessCard> createState() => SuccessCardPage();
+}
+class SuccessCardPage extends State<SuccessCard> {
```

Comment:
Looks like this component does not have a state, better in this case use StatelessWidget

Reply:
updated them to stateless widget